invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Profile/ProfileContainer.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Container } from 'semantic-ui-react';
+import {
+  PatronCurrentLoans,
+  PatronMetadata,
+  PatronPastLoans,
+  PatronPendingLoans,
+} from './components';
+import { sessionManager } from '../../../authentication/services';
+
+export default class ProfileContainer extends Component {
+  render() {
+    const currentUser = sessionManager.user;
+    return (
+      <Container>
+        <PatronMetadata username={currentUser.username} />
```

Comment:
I don't know if it's better to pass the prop down or just take it directly to each component. These components most probably will not be reused so I don't know if it makes sense to take the user prop.

Reply:
For the specific case, we can take it in the component directly, you are right, but in every other case the container has to pass down the data to the presentation components. Whatever you prefer for this one.

Reply:
the `currentUser` is very specific and as we can read it any time anywhere for logged user I think we should take it directly

Reply:
👍

Reply:
Well, an update on this one, this would work only for `PatronMetadata` for the rest of the components `PatronPendingLoans`, `PatronCurrentLoans`, `PatronPastLoans` since they are used elsewhere we need to provide the patronPid to maintain reusability.

Reply:
@topless But they are not as you created new ones no? The backoffice ones are different components. If it's confusing I would say to rename this to `CurrentPatronPendingLoans` or something that indicates that this is for the current user.

Reply:
I kept it as-is but made sure `patronPid` is a required prop for the components. I think it's fine to pass the prop even if it's always the current user. Generally makes it more testable to pass the prop.